README.md

```diff
@@ -54,28 +61,33 @@ _Require ssh tunnel to an ldap server and an WAR archive_
     SMTP_USER=user@jenkins.io
     SMTP_AUTH=true
     SMTP_PASSWORD=password
+    ```
+
+* Run docker-compose:
+
+```shell
+docker-compose up --build accountapp
 ```
-* Run docker-compose 
-```docker-compose up --build accountapp```
 
 ## Packaging
 
 For deploying to production, this app gets containerized. The container expects
 to see `/etc/accountapp` mounted from outside that contains the above mentioned
 `config.properties`
 
-
 To run the container locally, build it then:
 
-    docker run -ti --net=host  -v `pwd`:/etc/accountapp jenkinsciinfra/account-app:latest
+```shell
+docker run -ti --net=host  -v `pwd`:/etc/accountapp jenkinsciinfra/account-app:latest
```

Comment:
These backticks seem strange here to me (?)

Reply:
It is a shell script operator which spawns a subshell. 

But it is usually better to use `$()` operator instead (ref. https://www.redhat.com/sysadmin/backtick-operator-vs-parens).

Gotta update it.

Reply:
```suggestion
docker run -ti --net=host  -v "$(pwd)":/etc/accountapp jenkinsciinfra/account-app:latest
```